polyglot-chat-app/README.md

```diff
@@ -3,35 +3,36 @@
 This example demonstrates how to integrate Python on GraalVM with a Micronaut application.
 The application uses the Gradle build tool.
 
-### Prerequisites
-- [Native Image](https://www.graalvm.org/latest/reference-manual/native-image/)
-- [Python support](https://www.graalvm.org/latest/reference-manual/python/)
-
 ## Preparation
 
-1. Download the latest GraalPy as described on [https://www.graalvm.org/python/](https://www.graalvm.org/python/). For example on Linux:
-    ```bash
-    wget https://github.com/oracle/graalpython/releases/download/graal-23.1.1/graalpy-23.1.1-linux-amd64.tar.gz
-    tar xzf graalpy-23.1.1-linux-amd64.tar.gz
-    ```
+1. Download and install the latest GraalPy as described in the [Getting Started guide](https://www.graalvm.org/latest/reference-manual/python/#installing-graalpy). For example on Linux:
+   ```bash
+   wget https://github.com/oracle/graalpython/releases/download/graal-23.1.1/graalpy-23.1.1-linux-amd64.tar.gz
+   tar xzf graalpy-23.1.1-linux-amd64.tar.gz
+   ```
 
 2. Install the required packages for this demo into the _resources_ directory:
    ```bash
    graalpy-23.1.1-linux-amd64/bin/graalpy -m venv src/main/resources/venv
+   ```
+   ```bash
    src/main/resources/venv/bin/graalpy -m pip install nltk
    ```
+   ```bash
+   src/main/resources/venv/bin/graalpy -c "import nltk; nltk.download('vader_lexicon')"
+   ```
 
-3. Optional: Download and install GraalVM JDK for Java 21 or later to run Python with runtime compilation and to build a native image.
-   The demo will work on any OpenJDK distribution, but will be much faster on GraalVM JDK.
+3. (Optional) Download and install GraalVM JDK for Java 21 or later to run Python with runtime compilation and to build a native image. 
+The demo will work on any OpenJDK distribution, but will be much faster on GraalVM JDK.
 
-## Building and Running the application:
+## Building and Running the Application:
 
```

Comment:
I think there's a step missing here because gradle doesn't copy _src/main/resources/venv_ into the _target_ directory.

Reply:
Yes, indeed, if you want to run the native image you need to manually copy